implementations/rust/ockam/ockam_command/src/credential/issue.rs

```diff
@@ -0,0 +1,71 @@
+use std::collections::HashMap;
+
+use crate::{
+    identity::default_identity_name,
+    util::{node_rpc, print_encodable},
+    vault::default_vault_name,
+    CommandGlobalOpts, EncodeFormat, Result,
+};
+use anyhow::Context as _;
+use clap::Args;
+use ockam::Context;
+use ockam_identity::{credential::Credential, IdentityIdentifier};
+
+#[derive(Clone, Debug, Args)]
+pub struct IssueCommand {
+    #[arg(long = "as", default_value_t = default_identity_name())]
+    pub as_identity: String,
+
+    #[arg(long = "for", value_name = "IDENTITY_ID")]
+    pub for_identity: IdentityIdentifier,
+
+    /// Attributes in `key=value` format to be attached to the member
+    #[arg(short, long = "attribute", value_name = "ATTRIBUTE")]
+    pub attributes: Vec<String>,
+
+    #[arg(default_value_t = default_vault_name())]
+    pub vault: String,
+
+    /// Encoding Format
+    #[arg(long = "encoding", value_enum, default_value = "plain")]
+    encode_format: EncodeFormat,
+}
+
+impl IssueCommand {
+    pub fn run(self, opts: CommandGlobalOpts) {
+        node_rpc(run_impl, (opts, self));
+    }
+
+    fn attributes(&self) -> Result<HashMap<String, String>> {
```

Comment:
I would introduce a cli parser here to split keys/values so that `attributes` can be directly parsed as a `HashMap<String, String>`. Then we can reuse this kind of parser for other key/value settings on the command line

Reply:
For the sake of time, and trying to get this out the door. I'll push this into a github issue, this would also need to be updated for other commands (Enroll) as well.